integreat_cms/core/utils/word_count.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from html import unescape
+from typing import TYPE_CHECKING
+
+from django.utils.html import strip_tags
+
+if TYPE_CHECKING:
+
+    from ...cms.models import EventTranslation, PageTranslation, POITranslation
+
+
+def word_count(
+    translation: EventTranslation | (PageTranslation | POITranslation),
+) -> int:
+    """
+    This function counts the number of words in a content translation
+    """
+    attributes = [translation.title, translation.content]
+    if translation.foreign_field() == "poi":
+        # Currently only POI translations have a meta_description
+        attributes.append(translation.meta_description)
```

Comment:
I'd prefer to keep it like the previous version, where the code that includes the meta description did not check the type of the translation. This way, if we ever add meta_description to other types, we will not have to update this function

```suggestion
    attributes = [
            getattr(source_translation, attr, None)
            for attr in ["title", "content", "meta_description"]
    ]
```
Alternatively, word count could only take a string as a parameter and leave that logic to the callers, but I am fine with either approach.